apps/condo/domains/user/schema/GetAccessTokenByUserIdService.js

```diff
@@ -39,7 +40,11 @@ const GetAccessTokenByUserIdService = new GQLCustomSchema('GetAccessTokenByUserI
     types: [
         {
             access: true,
-            type: 'input GetAccessTokenByUserIdInput { userId: ID!, type: String! }',
+            type: 'input GetAccessTokenByUserIdInput { userId: UserWhereUniqueInput!, type: GetAccessTokenByUserIdServiceType!, organization: OrganizationWhereUniqueInput }',
+        },
+        {
+            access: true,
+            type: `enum GetAccessTokenByUserIdServiceType { ${SBBOL_IDENTITY_TYPE} }`,
```

Comment:
so that there are no misconceptions in the listing. If there is an enum containing all the integrations from the statement, then you might think that the token can be obtained using this mutation, but without adding a condition to the code to search for the token from the required integration, nothing will work. So, so that the application types do not contradict the behavior, I think it is better to write all available integrations in the mutation type